src/main/java/io/vertx/java/spi/cluster/impl/jgroups/services/DefaultRpcExecutorService.java

```diff
@@ -80,13 +79,16 @@ public DefaultRpcExecutorService(RpcDispatcher dispatcher) {
     try {
       NotifyingFuture<RspList<T>> notifyingFuture = this.<T>execute(action, options);
       notifyingFuture.setListener((future) -> {
-        try {
-          RspList<T> rspList = future.get();
-          T result = futureDone(rspList);
-          handler.handle(Future.succeededFuture(result));
-        } catch (Exception e) {
-          handler.handle(Future.failedFuture(e));
-        }
+        vertx.executeBlocking(
```

Comment:
I don't understand this - isn't the whole point of a future listener that it's not called until the result is ready? So why use executeBlocking for the get()?